pkg/apis/machineconfiguration.openshift.io/v1/register.go

```diff
@@ -35,8 +34,6 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&MachineConfigList{},
 		&MachineConfigPool{},
 		&MachineConfigPoolList{},
-		&osev1.Node{},
-		&osev1.NodeList{},
```

Comment:
You shouldn't be adding resources from other groups to the scheme here, it causes confusion for the decoder (and was breaking/masking issues in tests). The correct thing to fix this is `configv1.Install(scheme.Scheme)` at some point in the process that was complaining